src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { navigationItems } from '../../constants/Navigation';
+import { NavigationIcon } from '../NavigationIcon/NavigationIcon';
+import { NavigationIconLink } from '../NavigationIconLink/NavigationIconLink';
+import './Navigation.scss';
+
+export const Navigation: React.FC = () => (
+  <nav className="navigation">
+    <ul className="navigation__ul">
```

Comment:
ul refers to the HTML which could change. It should refer to the actual thing -> it just list or wrapper so it could be navigation__wrapper or list